lib/ProMotion/table/table.rb

```diff
@@ -118,8 +118,7 @@ def accessory_toggled_switch(switch)
 
     def delete_row(index_paths, animation = nil)
       deletable_index_paths = []
-      index_paths = [index_paths] if index_paths.kind_of?(NSIndexPath)
-      index_paths.each do |index_path|
+      Array(index_paths).each do |index_path|
```

Comment:
`index_paths.kind_of?(NSIndexPath)` doesn't work anymore. It's returning an `ImmediateRef` object. I'm filing a support ticket, since Laurent said it should still work as we expected.

This is a better way to do it anyway.